.travis.yml

```diff
@@ -5,7 +5,7 @@ scala:
   - 2.12.6
   - 2.13.0-M4
 jdk:
-  - openjdk7
+  - openjdk6
```

Comment:
i think we still need the explicit `jdk_switcher` command, no? https://github.com/scala/scala-collection-compat/commit/f43e7725b57fe1688a9024bba3d6cfe0e712304d#diff-354f30a63fb0907d4ad57269548329e3L48